packages/plugin-vue/src/handleHotUpdate.ts

```diff
@@ -212,6 +212,11 @@ function deepEqual(obj1: any, obj2: any, excludeProps: string[] = []): boolean {
     return obj1 === obj2
   }
 
+  // Check if both equals object
```

Comment:
This can and should be moved to the very top, no?

Reply:
Agreed, corrected